util/storage-samples/storage-owner-sample/src/main/java/org/fidoalliance/fdo/storage/OwnerDbManager.java

```diff
@@ -592,7 +594,8 @@ public Composite getSystemResources(DataSource ds,
         + "CONTENT "
         + "FROM SYSTEM_MODULE_RESOURCE "
         + "WHERE (RESOURCE_ID = ? AND CONTENT IS NOT NULL) OR "
-        + "(RESOURCE_ID = (SELECT CONTENT_RESOURCE_TAG WHERE RESOURCE_ID = ?))";
+        + "(RESOURCE_ID = (SELECT SELECT CONTENT_RESOURCE_TAG FORM SYSTEM_MODULE_RESOURCE WHERE "
```

Comment:
This was an attempt at a AIO fix.  There are a few other AIO fixes that I need to post so I will take take of this in a separate PR.  Meanwhile nothing will change since the original SQL was invalid and the new one is as well.